modules/customizing-parameters-serving-runtime.adoc

```diff
@@ -0,0 +1,37 @@
+:_module-type: PROCEDURE
+
+[id="customizing-parameters-serving-runtime{context}"]
+= Customizing the parameters of a deployed model-serving runtime
+
+[role='_abstract']
+You can modify the parameters of an existing runtime to suit your business requirements.
+
+NOTE: Customizing the parameters of a runtime only affects the model deployment selected.
```

Comment:
Suggestion:
_only affects the selected model deployment_ OR _affects the model deployment that you selected_.